recipes/react-router/app/routes/puck-splat.tsx

```diff
@@ -40,9 +40,9 @@ export async function loader({ params }: Route.LoaderArgs) {
 export function meta({ data: loaderData }: Route.MetaArgs) {
   return [
     {
-      title: loaderData.isEditorRoute
+      title: loaderData?.isEditorRoute
```

Comment:
```suggestion
      title: loaderData.isEditorRoute
```

The loader should always return an object for `loaderData`, so the optional chaining here shouldn’t be necessary.